sites/partners/.env.template

```diff
@@ -12,6 +12,8 @@ SHOW_SMS_MFA=TRUE
 # YYYY-MM-DD HH:mm Z,YYYY-MM-DD HH:mm Z
 # 2024-02-05 18:00 -08:00,2024-02-05 18:55 -08:00
 MAINTENANCE_WINDOW=
+SITE_MESSAGE_WINDOW=
```

Comment:
nit: doesn't look like this is setup for partners as of now. So we don't need it in this template file

Reply:
Removed!